spec/scope_spec.rb

```diff
@@ -76,5 +78,15 @@
         expect(subject.include?(unknown)).to eq(true)
       end
     end
+
+    context "for ignored values" do
+      subject { described_class.new([value], ignore: [value]) }
+
+      let(:value) { Ronin::Recon::Values::IP.new('1.2.3.4') }
```

Comment:
Use a different name here so the `subject` can be initialized with `values` and `ignored_values`, and the specs can test that un-ignored values return `true` and ignored values return `false`.